controllers/volumesnapshot.go

```diff
@@ -38,6 +38,11 @@ func (r *VolumeSnapshotBackupReconciler) MirrorVolumeSnapshotContent(log logr.Lo
 		return false, err
 	}
 
+	if vscInCluster.Status == nil || vscInCluster.Status.ReadyToUse == nil || *vscInCluster.Status.ReadyToUse != true {
```

Comment:
have nil status checks a separate conditional to explicitly mention status isn't set.

if `ReadyToUse != true` should be the conditional that logs that its not ready